test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -217,22 +222,21 @@
                                                        "foo-2" "bar"}))
 
       ;; Add two facts with the same :path, but different values.
-      (let [[p1 p2] (map :id (sql/insert-records
-                              :fact_paths
-                              {:value_type_id 0
-                               :path "orphan" :name "orphan" :depth 0}
-                              {:value_type_id 1
-                               :path "orphan" :name "orphan" :depth 0}))]
-        (sql/insert-records
-         :fact_values
-         {:path_id p1
-          :value_type_id 0
-          :value_hash (hash/generic-identity-hash "1")
-          :value_string "1"}
-         {:path_id p2
-          :value_type_id 1
-          :value_hash (hash/generic-identity-hash 1)
-          :value_integer 1}))
+      (let [[p1 p2]
+            (map :id (sql/insert! (jdbc/db) :fact_paths
+                                  {:value_type_id 0
+                                   :path "orphan" :name "orphan" :depth 0}
+                                  {:value_type_id 1
+                                   :path "orphan" :name "orphan" :depth 0}))]
+        (sql/insert! (jdbc/db) :fact_values
```

Comment:
Let's chat about this - I've gotten the impression there are varying opinions (though none categorical), and I started these changes accordingly, but as a practical matter I suspect we might be better off if I rework this (perhaps more in the direction you suggest).